doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -78,145 +78,107 @@ Displaying
 Flags, Options and Tables
 -----------------------------
 
-|Coq| configurability is based on flags (e.g. :opt:`Printing All`), options
-(e.g. :opt:`Printing Width`), or tables (e.g. :cmd:`Add Printing Record`). The
-names of flags, options and tables are made of non-empty sequences of
-identifiers (conventionally with capital initial letter). The general commands
-handling flags, options and tables are given below.
+Coq has many settings to control its behavior.  Setting types include flags, options
+and tables:
 
-.. TODO : flag is not a syntax entry
+* A :production:`flag` has a boolean value, such as :flag:`Asymmetric Patterns`.
+* An :production:`option` has a numeric or string value, such as :opt:`Firstorder Depth`.
+* A :production:`table` contains a set of strings.
+* In addition, some commands provide settings, such as :cmd:`Extraction Language OCaml`.
 
-.. cmd:: Set @flag
+.. FIXME Convert `Extraction Language OCaml` to an option.
```

Comment:
One possibility is that support for declaring new types of options is added to `library/goptions.ml`.

The FIXME should not stay in the reference manual in any case.

Reply:
`FIXME` doesn't appear in the document; above @Zimmi48 is asking me to add `FIXME` comments.

Are you OK with that?

Reply:
Ah sorry, I did not realize that `FIXME` was a sphinx keyword!

Reply:
Not a keyword but Sphinx comments because of the `..` and the fact it is not a known directive.